src/index.js

```diff
@@ -1 +1,3 @@
 import webex from 'webex';
+import WebexSDKAdapter from '@webex/sdk-component-adapter';
```

Comment:
Can you add a comment as to you you're adding this?
```suggestion
// Testing bundle works with component libraries
import WebexSDKAdapter from '@webex/sdk-component-adapter';
```